crates/cxx-qt-gen/src/generator/cpp/inherit.rs

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+
+use indoc::formatdoc;
+
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
seems this has been moved down somehow?

Reply:
Ah, rust-analyzers `auto-use` doing it's thing again :see_no_evil: